sys/net/if_bridge.c

```diff
@@ -3643,6 +3650,7 @@ bridge_pfil(struct mbuf **mp, struct ifnet *bifp, struct ifnet *ifp, int dir)
  * XXX Right now we update ipstat but not
  * XXX csum_counter.
  */
+#ifdef INET
```

Comment:
IMO the ifdef should come before the comment which applies to the conditionally compiled code.

Reply:
fixed.